compiler/resolution/functionResolution.cpp

```diff
@@ -6914,9 +7226,7 @@ requiresImplicitDestroy(CallExpr* call) {
 
         // Return type is a record (which includes array, record, and
         // dist) or a ref counted type that is passed by reference
```

Comment:
It refers to ref counted types passed by ref, but no longer makes the ref or refCountedType check